src/components/complete-profile/CompleteProfile.js

```diff
@@ -0,0 +1,65 @@
+import { useTranslation } from 'react-i18next'
+import { Accordion, AccordionDetails, AccordionSummary, Box, IconButton, Typography } from '@mui/material'
+import AppProgressBar from '../app-progress-bar-line/AppProgressBarLine.js'
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
+import ExpandLessIcon from '@mui/icons-material/ExpandLess'
+import { styles } from './CompleteProfile.styles.js'
+import { useMemo, useState } from 'react'
+import ProfileItem from '../profile-item/ProfileItem.js'
+import { Link, useLocation } from 'react-router-dom'
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward'
+
+const CompleteProfile = ({ profileItems, data, expanded = false }) => {
```

Comment:
i don't see where do you use expanded prop now. why do you need it?

Reply:
I thought it would be needed when we go from the home page to the profile page. And after that progress bar will be open.